<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMapUnitScaleDialog</class>
 <widget class="QDialog" name="QgsMapUnitScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust scaling range</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/themes/default/mActionOptions.png</normaloff>:/images/themes/default/mActionOptions.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="mCheckBoxMaxScale">
     <property name="text">
      <string>Maximum scale:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mCheckBoxMinScale">
     <property name="text">
      <string>Minimum scale:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QgsScaleComboBox" name="mComboBoxMaxScale"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QgsScaleComboBox" name="mComboBoxMinScale"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Scale only within the following map unit scale range:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsScaleComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsscalecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCheckBoxMaxScale</tabstop>
  <tabstop>mComboBoxMaxScale</tabstop>
  <tabstop>mCheckBoxMinScale</tabstop>
  <tabstop>mComboBoxMinScale</tabstop>
  <tabstop>mButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsMapUnitScaleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsMapUnitScaleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
